core/metacling/src/TCling.cxx

```diff
@@ -2888,6 +2888,18 @@ Bool_t TCling::IsLoaded(const char* filename) const
    return kFALSE;
 }
 
+Bool_t TCling::IsLibraryLoaded(const char* libname) const
+{
+   R__LOCKGUARD(gInterpreterMutex);
+
+   // Check shared library.
+   TString tlibname(libname);
+   const char *found = gSystem->FindDynamicLibrary(tlibname, kTRUE);
+   if (found)
+      return fInterpreter->getDynamicLibraryManager()->isLibraryLoaded(libname);
```

Comment:
You probably want to pass `tlibname` to `isLibraryLoaded()`? Maybe rename `tlibname` to `fullLibName` to make it visually more distinguishable from `libname`?